### PR TITLE
update the API version

### PR DESCRIPTION
**Pull request by mumian**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by mumian** [comment]
@bmoore-msft - i have updated the API version, and made some changes to the template.  I am not sure about the principalID value in the parameter file., so I use the old value.

File: 101-rbac-builtinrole-resourcegroup/README.md [hunk 1]
```diff
@@ -10,22 +10,43 @@ This template assigns Owner, Reader or Contributor access to an existing resourc
 - Principal ID
 - Role Definition Type
 
-
-**Use following powershell command to get Principal ID associated with a user using their email id. Please note, principal id maps to the id inside the directory and can point to a user, service principal, or security group. The ObjectId is the principal ID.
+**Use following Azure PowerShell cmdlet to get Principal ID associated with a user using their email id. Please note, principal id maps to the id inside the directory and can point to a user, service principal, or security group. The ObjectId is the principal ID.
```
Comment (bmoore-msft) [inline 101-rbac-builtinrole-resourcegroup/README.md]: Use the following...

File: 101-rbac-builtinrole-resourcegroup/README.md [hunk 2]
```diff
@@ -10,22 +10,43 @@ This template assigns Owner, Reader or Contributor access to an existing resourc
...
 
     PS C:\> Get-AzureRmADUser -mail <email id>
     
     DisplayName                    Type                           ObjectId
     -----------                    ----                           --------
     <NAME>                                                        xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
 
+**Use following Azure PowerShell cmdlet to learn about RoleDefinitions.
```
Comment (bmoore-msft) [inline 101-rbac-builtinrole-resourcegroup/README.md]: Use the following...

File: 101-rbac-builtinrole-resourcegroup/README.md [hunk 3]
```diff
@@ -10,22 +10,43 @@ This template assigns Owner, Reader or Contributor access to an existing resourc
...
+    Id               : acdd72a7-3385-48ef-bd42-f606fba81ae7
+    IsCustom         : False
+    Description      : Lets you view everything, but not make any changes.
+    Actions          : {*/read}
+    NotActions       : {}
+    DataActions      : {}
+    NotDataActions   : {}
+    AssignableScopes : {/}
+
+   You can use the same cmdlet to get the role definition ID for owner, and contributor.
```
Comment (bmoore-msft) [inline 101-rbac-builtinrole-resourcegroup/README.md]: remove comma

File: 101-rbac-builtinrole-resourcegroup/azuredeploy.json [hunk 4]
```diff
@@ -2,45 +2,34 @@
...
   "parameters": {
-    "principalId": {
+    "roleAssignmentName": {
       "type": "string",
       "metadata": {
-        "description": "The principal to assign the role to"
+        "description": "Specifies the name of the role assignment to create. It can be any valid GUID."
       }      
     },
-    "builtInRoleType": {
```
Comment (bmoore-msft) [inline 101-rbac-builtinrole-resourcegroup/azuredeploy.json]: changing this behavior makes the readme inaccurate - need to reconcile that...

File: 101-rbac-builtinrole-resourcegroup/README.md (hunk 1 above)
Comment (mumian) [reply 101-rbac-builtinrole-resourcegroup/README.md]: Fixed.

File: 101-rbac-builtinrole-resourcegroup/README.md (hunk 2 above)
Comment (mumian) [reply 101-rbac-builtinrole-resourcegroup/README.md]: Fixed.

File: 101-rbac-builtinrole-resourcegroup/README.md (hunk 3 above)
Comment (mumian) [reply 101-rbac-builtinrole-resourcegroup/README.md]: Fixed.

File: 101-rbac-builtinrole-resourcegroup/azuredeploy.json (hunk 4 above)
Comment (mumian) [reply 101-rbac-builtinrole-resourcegroup/azuredeploy.json]: Fixed.

**Pull request merged**